{{cookiecutter.repostory_name}}/noxfile.py

```diff
@@ -126,7 +126,7 @@ def lint(session):
     session.run("codespell", ".")
     run_shellcheck(session, mode="check")
     run_readable(session, mode="check")
-    session.run("ruff", "format", "--check", ".")
+    session.run("ruff", "format", "--diff", ".")
```

Comment:
please add this line to https://github.com/reef-technologies/cookiecutter-rt-django/blob/master/noxfile.py#L121 as well

Reply:
Added that, formatted, also added `.git-blame-ignore-revs`.